app/assets/javascripts/views/shared/footer.js

```diff
@@ -0,0 +1,35 @@
+(function (App) {
+  App.View.Footer = Backbone.View.extend({
+
+    el: '.c-footer',
+
+    initialize: function () {
+      this._setVars();
+      this._setEventListeners();
+    },
+
+    _setVars: function () {
+      this.body = document.querySelector('body');
+      this.refElem = this.el.querySelector('.js-toggle-tooltip');
+      this.socialTooltip = new App.View.SocialTooltip({ refElem: this.refElem });
+    },
+
+    _setEventListeners: function () {
+      this.refElem.addEventListener('mouseenter', function () {
+        if (this.body.contains(this.socialTooltip.el)) {
+          if (this.socialTooltip.el.classList.contains('_is-hidden')) {
```

Comment:
This code should be contained in the component. From outside, you don't know which class is used. Create a new method `isHidden` (or whatever name) in the component for that.